doc/manual/src/glossary.md

```diff
@@ -56,6 +64,11 @@
   A derivation which includes the
   [`outputHash`](./language/advanced-attributes.md#adv-attr-outputHash) attribute.
 
+  In package definitions such as in [nixpkgs], fixed-output derivations are
+  commonly created by [fetchers].
+
+  [fetchers]: https://nixos.org/manual/nixpkgs/stable/#chap-pkgs-fetchers
+
```

Comment:
It doesn't specifically have to be a reference to Nixpkgs, however as you point out in that linked section the builtin fetchers don't create fixed-output derivations (something I wasn't 100% sure of until reading that section, thank you for that!), so they can't be used to provide context on why a fixed-output derivation matters or where it might come up.

Perhaps it could be mentioned that fixed-output derivations allow network access in the sandbox? (If that's even true; I assume that it is but didn't include it here since I couldn't find anything else documenting that)

Reply:
FODs are only okay-documented, and I suggest not trying to solve that right here. We can add bits and pieces independently until someone runs into the urge of adding or rewriting a section.